translations/de.ts

```diff
@@ -1909,6 +1884,86 @@ Viele Grüße,
         <comment>Desktop as a camera input for screen sharing</comment>
         <translation>Bildschirm</translation>
     </message>
+    <message>
+        <source>Incorrect response</source>
+        <translation>Unkorrekte Antwort.</translation>
+    </message>
+    <message>
+        <source>No password in response</source>
+        <translation>Kein Passwort als Antwort</translation>
+    </message>
+    <message>
+        <source>Server doesn&apos;t support Toxme</source>
+        <translation>Server unterstützt kein ToxMe.</translation>
+    </message>
+    <message>
+        <source>You must send POST requests to /api</source>
+        <translation>Du musst POST-Anfragen an /api senden.</translation>
+    </message>
+    <message>
+        <source>Please try again using a HTTPS connection</source>
+        <translation>Bitte versuche es noch einmal mit einer HTTPS-Verbindung.</translation>
+    </message>
+    <message>
+        <source>I was unable to read your encrypted payload</source>
+        <translation>Ich konnte deinen verschlüsselten Payload nicht lesen.</translation>
+    </message>
+    <message>
+        <source>You&apos;re making too many requests. Wait an hour and try again</source>
+        <translation>Du machst zu viele Anfragen. Bitte warte eine Stunde und versuche es dann erneut.</translation>
+    </message>
+    <message>
+        <source>This name is already in use</source>
+        <translation>Dieser Name ist bereits in Verwendung.</translation>
+    </message>
+    <message>
+        <source>This Tox ID is already registered under another name</source>
+        <translation>Diese Tox ID ist bereits unter einem anderen Namen registriert.</translation>
+    </message>
+    <message>
+        <source>Please don&apos;t use a space in your name</source>
+        <translation>Dein Name darf keine Leerzeichen beinhalten.</translation>
+    </message>
+    <message>
+        <source>Password incorrect</source>
+        <translation>Falsches Passwort.</translation>
+    </message>
+    <message>
+        <source>You can&apos;t use this name</source>
+        <translation>Du kannst diesen Namen leider nicht verwenden.</translation>
+    </message>
+    <message>
+        <source>Name not found</source>
+        <translation>Name nicht gefunden.</translation>
+    </message>
+    <message>
+        <source>Tox ID not sent</source>
+        <translation>Tox ID wurde nicht gesendet.</translation>
+    </message>
+    <message>
+        <source>Lookup failed because the server replied with invalid data</source>
+        <translation>Lookup ist gescheitert, weil die Server-Antwort ungültig war.</translation>
+    </message>
+    <message>
+        <source>That user does not exist</source>
+        <translation>Dieser Benutzer existiert nicht.</translation>
```

Comment:
You're right. The English strings are not perfect in some cases.
So sometimes the German translation is a little bit different / better as the original source.

There was a discussion about that topic a few months ago and the consensus was that we'll make a good German translation and this could be a potential template for the English language someday. But it doesn't make sense to change it now because there will be many change in the future as well - when qTox is more stable and matured, the fine-tuning for the language/translation makes sense. But at the moment, we should focus on other things.